ERCS/erc-xxxx.md

```diff
@@ -0,0 +1,243 @@
+
+Returns:
+
+1. Object
+
+| Name                        | Type     |
+|-----------------------------|----------|
+| validationGasLimit          | QUANTITY |
+| paymasterValidationGasLimit | QUANTITY |
+| paymasterPostOpGasLimit     | QUANTITY |
+| callGasLimit                | QUANTITY |
+
+Error:
+
+DATA - The revert data of the first reverted frame.
+CODE - The error code indicating the type of error, which may include the entity that caused the revert on-chain.
+MESSAGE - The human-readable error that may include a decoding of the `DATA` field if possible.
+
+#### Notes on implementation details
+
+As mentioned in the RIP-7560, the `sender` and `paymaster` contracts should not revert on the validation failure
+and should make calls to `sigFailAccount` and `sigFailPaymaster` respectively
+in order to support `eth_estimateRip7560TransactionGas`.
+
+The recommended way to achieve this behavior for Smart Contract Accounts and Paymasters is to compare the `signature`
+parameter to a predetermined "dummy signature" and to call a `sigFail` callback in case the values match.
+
+### Create a new JSON-RPC API method `eth_traceRip7560Validation`
+
+Only executes the validation phase of the RIP-7560 transaction and returns the tracing results of this execution.
+This is done in order to allow other clients to determine
+whether all contracts used within the validation phase of this transaction are compliant with the rules
+defined in the [ERC-7562](https://eips.ethereum.org/EIPS/eip-7562).
+
+Parameters:
+
+1. OBJECT - The RIP-7560 transaction object.
+2. QUANTITY | TAG - a block number, or the string "earliest", "latest", "pending", "safe" or "finalized", as in the
+   default block parameter.
+
+Returns:
+
+1. OBJECT - the tracing result of executing the entire validation phase of the RIP-7560 transaction.\
+The exact shape of the returned object is not strictly defined by this standard and can be decided
+by different implementation.
+
+### Error Codes
+
+* code: -32500 - transaction validation failed by `sender`.
+  The message field SHOULD be set to the revert message and data from the `sender`.
+
+* code: -32501 - transaction validation failed by `paymaster`.
+  The message field SHOULD be set to the revert message and data from the `paymaster`.
+
+* code: -32502 - transaction validation failed by `deployer`
+  The message field SHOULD be set to the revert message and data from the `deployer`.
+
+* code: -32503 - Transaction out of time range.
+  The message field SHOULD include the requested time range and the current block timestamp.
+
+## Rationale
+
+### Creating `eth_executeRip7560Transaction` instead of modifying `eth_call`
+
+The semantics of the `eth_call` are currently very simple for all existing transaction types, and would become
+significantly more complex with the addition of the RIP-7560 transaction type support.
+
+It seems like the difference between these transaction types warrants a separate standalone RPC API method.
```

Comment:
I agree. But where will it be defined if not here?

Reply:
This comment relates to the `eth_executeRip7560Transaction` API that is defined in this ERC.